backend/lib/services/terminals/index.js

```diff
@@ -409,23 +396,19 @@ async function getOrCreateTerminalSession ({ user, namespace, name, target, body
     throw new Error('Hosting cluster or target cluster is hibernated')
   }
 
-  const dashboardClient = GardenerDashboard(user)
-  const gardenCoreClient = Core(user)
-
-  const hostKubeconfig = await getKubeconfig({ coreClient: gardenCoreClient, ...hostCluster.secretRef })
+  const hostKubeconfig = await client.getKubeconfig(hostCluster.secretRef)
```

Comment:
use createHostClient?

Reply:
The hostKubeconfig is not used later. We do not need a client, don't we?

Reply:
Confirmed with @petersutter that no action is needed.
✔️